tests/integration/base_object_test.go

```diff
@@ -37,8 +37,8 @@ var (
 )
 
 // Smoke Test for ObjectStore - Test check the following operations on ObjectStore in order
-//Create object store, Create User, Connect to Object Store, Create Bucket, Read/Write/Delete to bucket,Delete Bucket and
-//Delete user
+//Create object store, Create User, Connect to Object Store, Create Bucket, Read/Write/Delete to bucket,
```

Comment:
Add a space in the beginning of this line 40. It should be on the same level as 39 and 41.